### PR TITLE
Add artifact information to compose.complete message

### PR DESCRIPTION
**Pull request by puiterwijk**
Signed-off-by: Patrick Uiterwijk puiterwijk@redhat.com

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
Looks good to me :+1:. Need to remove [L303](https://github.com/kushaldas/autocloud/pull/50/files#diff-f4b9670d7e11088c311dc4c184c8e8b0R303)

**Comment by puiterwijk** [comment]
Line 303 removed

File: autocloud_job.py [hunk 1]
```diff
@@ -247,33 +247,39 @@ def check_status_of_compose_image(compose_id):
...
+            'family': compose_job_obj.family.value,
+            'image_url': compose_job_obj.image_url,
+            'release': compose_job_obj.release,
+            'subvariant': compose_job_obj.subvariant,
+            'format': compose_job_obj.image_format,
+            'type': compose_job_obj.image_type,
+            'name': compose_job_obj.image_name,
+            'status': compose_job_obj.status.value
+        }
+        results['artifacts'][compose_job_obj] = artifact
```
Comment (sayanchowdhury) [inline autocloud_job.py]: `compose_job_obj` needs to be replaced by string instead of object.

File: autocloud_job.py [hunk 2]
```diff
@@ -336,6 +341,5 @@ def main():
         result, running_status = auto_job(task_data)
```
Comment (sayanchowdhury) [inline autocloud_job.py]: Need to drop `running_status` here.

**Comment by puiterwijk** [comment]
Fixed.

**Pull request merged**